datasets.py

```diff
@@ -266,11 +266,11 @@ def from_path(self, file_path):
         scene_group = self.scene_mapping[file_path] # optional
 
         #optional. lat_lng is a dictionary with 2 keys: 'lat' and 'lng'
-        # whose values are doubles
+        # whose values are doubles. 
```

Comment:
specify "type doubles"? otherwise might be confusing that lat is double as in two numbers or something